plugins/pulp_ostree/plugins/importers/steps.py

```diff
@@ -34,29 +36,46 @@ def __init__(self, **kwargs):
         self.remote_id = model.generate_remote_id(self.feed_url)
         self.storage_path = os.path.join(
             constants.SHARED_STORAGE, self.remote_id, constants.CONTENT_DIR)
+        self.repo_id = self.get_repo().id
         self.add_child(Create())
         self.add_child(Pull())
         self.add_child(Add())
+        self.add_child(Clean())
 
 
 class Create(PluginStep):
     """
     Ensure the local ostree repository has been created
-    and the configured.
+    and the configured.  A temporary remote is created using the repo_id as
+    the remote_id.  The remote is created using the Remote which stores SSL
```

Comment:
Maybe note that this is done to avoid conflict with other local repos tracking the same feed? I spent a lot of time wondering *why* this temporary remote is made until I went back and read the PR comment (my own fault, admittedly).